app/services/hot_lunch/sum_family_cru_stu_cost.rb

```diff
@@ -0,0 +1,17 @@
+class HotLunch::SumFamilyCruStuCost < ChargesService
+  attr_accessor :family
+
+  def call
+    family_costs.each do |cost|
+      self.total_adjustments += defined?(cost.total_adjustments) ? cost.total_adjustments : Money.empty
+      self.subtotal += defined?(cost.subtotal) ? cost.subtotal : Money.empty
+      self.total += defined?(cost.total) ? cost.total : Money.empty
+    end
+  end
+
+  private
+
+  def family_costs
+    family.children.map { |c| HotLunch::ChargeChildCost.call(child: c) if c.crustu_grade?}
```

Comment:
Not very DRY when the only difference between these classes is this one line...

Reply:
Sorry accidental click.  Yes, I know, its the same for all of the hot lunch calculations.  I am leaving in a few hours for the rest of the week, and I only had an hour or so early this am to work on this for Andy and between my babies surgery the last week and my wife's cardiac appointments and stress tests, time was limited.  We can mark this in a TODO to avoid the code duplication.  This month I have to do the upgrade etc, so I can put this on the list for the todo.  WDYT?  I have a minor adjustment to make now for totals, so you can wait to approve.